### ISSUE TITLE
Explicitly include common/recommended element identifiers of compartments/parameters/species in ontology

### ISSUE THREAD

**Issue by jwkruisselbrink** [InSilicoVida-Research-Lab/pbpko#16]
It would be very useful if "common and/or recommended" element identifiers could be included (explicitly) as part of the ontology.

E.g., to use "Falv" as recommended element id for "alveolar ventilation rate". Or to use "pc_" as prefix for partition coefficients.

**Comment by Crispae** [comment]
@jwkruisselbrink as i understand, you are suggesting to add annotation property for common and recommended terms ?
such as using following annotation term [editor preferred term](http://purl.obolibrary.org/obo/IAO_0000111)

**Comment by jwkruisselbrink** [comment]
No. According to the description of the editor preferred term it is the "concise, meaningful, and human-friendly name for a class or property preferred by the ontology developers". I think that for this, the main labels should suffice.

At present, many of the labels are somewhat cryptic/non-descriptive because they seem to be based on element names of model implementations. As mentioned in issue #10, I think the labels should be more descriptive. However, the current names of the terms (which, in fact, seem to stem from some naming scheme in model implementations) could still be included in the ontology as "common element names in model implementations". In this case, it is not so much a "preferred term of ontology developers", but more related to the element names in model implementations.

For example, for the alveolar ventilation rate, the main label should be "alveolar ventilation rate", and one of the "commonly used parameter names" could be "Falv".

Hence, in this case, you may want to have a property "has_common_element_names" linking to possibly multiple commonly use element names.

**Comment by Crispae** [comment]
@jwkruisselbrink  in latest release [v1.1.0](https://github.com/InSilicoVida-Research-Lab/pbpko/releases/tag/v1.1.0) we try to address this, by making the label more descriptive, while also adding an annotation `alternative term` from `IAO` ontology. 

Please, let me know if doesn't meet the standard criteria, we can reopen this issue
